README.md

```diff
@@ -1,7 +1,7 @@
 # Python Session
 > Author: Daniel Huri
 > 
-> This was Built in Python 3.6+
```

Comment:
It was actually written using python 2.7

Reply:
Yeah, my bad, I saw prints with parenthesis (`(`, `)`) so I assumed it was python 3.6

Reply:
It fixed in the second commit...

Reply:
Maybe I needed to [amend](
https://www.atlassian.com/git/tutorials/rewriting-history) both commits